src/routes/graphql/resolvers.ts

```diff
@@ -82,6 +83,18 @@ export function resolveTxBundledIn(tx: GqlTransaction) {
   };
 }
 
+export async function resolveTxSignature(tx: GqlTransaction) {
+  if (tx.signature !== null) {
+    return tx.signature;
+  }
+
+  if (tx.parentId !== null) {
+    return signatureFetcher.getDataItemSignature(tx.id);
+  }
+
+  return '';
```

Comment:
return '<not-found>' instead

Reply:
Fixed here [`05a68d4` (#182)](https://github.com/ar-io/ar-io-node/pull/182/commits/05a68d4c07097d5dde47e5333ed6609ed3ad83be)